SignaturePad-Example/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion project.ANDROID_BUILD_TOOLS_VERSION
 
   defaultConfig {
-        minSdkVersion 9
+        minSdkVersion 14
```

Comment:
Why are you bumping up the minSdkVersion if there are no new features without backward compatibility?